### PR TITLE
add semantic-metrics implementation 

### PR DESCRIPTION
**Pull request by mattnworb**
this is the same as #86 except the PR is from a branch in this repo rather than a fork.

### CODE REVIEW

**Comment by mattnworb** [comment]
Hm, my theory was wrong. The travis build is sending data to coveralls ok, but it just has 0 lines / 0 coverage since cobertura is broken on Java 8.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+### 1.1.0
+Add `folsom-semantic-metrics` module containing a [semantic-metrics][]-based implementation of Folsom's `Metrics`.
```
Comment (spkrka) [inline CHANGELOG.md]: I don't think we need a new release for this

**Review by spkrka — APPROVED** [review]
Looks good, but I don't think we need to release 1.1.0 just for this - We can mvn deploy 1.0.0 of folsom-semantic-metrics separately.

File: CHANGELOG.md (hunk 1 above)
Comment (mattnworb) [reply CHANGELOG.md]: I can change this, but do you think it’s still useful to mention this in the changelog somewhere?

**Comment by mattnworb** [comment]
@spkrka I dropped the commit that had bumped the version, and uploaded [folsom-semantic-metrics 1.0.0 to maven central](https://oss.sonatype.org/#nexus-search;quick~folsom-semantic-metrics).

**Pull request merged**